lsdcGui.py

```diff
@@ -434,6 +443,14 @@ def robotOnCheckCB(self,state):
       else:
         setBlConfig("robot_online",0)
 
+    def beamCheckOnCheckCB(self,state):
+      if state == QtCore.Qt.Checked:
+         setBlConfig("beamCheck",1)
+         logger.debug("beamcheck on")
```

Comment:
for consistency, make capitalization the same for the variable as in the message

Reply:
I can't believe I missed that. Will change.